stack-templates.json

```
@@ -95,6 +95,21 @@
     "title": "MariaDB v10.3",
     "type": 2
   },
+  {
+    "categories": [
+      "database"
+    ],
+    "description": "MariaDB GALERA NEW",
```

maybe we can have a more descriptive name here?
```suggestion
    "description": "MariaDB Galera v25",
```

(same way as for e.g. Postgresql Stacks)